src/client/components/login-form.js

```diff
@@ -0,0 +1,100 @@
+import React, { Component } from "react";
+import { Route, Redirect } from "react-router-dom";
+
+class LoginForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      email: "",
+      password: "",
+      isLoggedin: false
+    };
+  }
+
+  clickHandlerCancel = event => {
+    this.setState({
+      email: "",
+      password: ""
+    });
+  };
+
+  handleChange = event => {
+    this.setState({
+      [event.target.name]: event.target.value
+    });
+  };
+
+  handleSubmit = event => {
+    event.preventDefault();
+
+    try {
+      // TODO: Add environment variable
+      fetch("api/auth/login", {
+        method: "POST",
+        headers: {
+          Accept: "application/json text/plain",
+          "Content-Type": "application/json"
+        },
+        body: JSON.stringify({
+          email: this.state.email,
+          password: this.state.password
+        })
+      })
+        .then(response => response.json())
+        .then(data => {
+          localStorage.setItem("token", data.token); //storing token into local storage
+          this.setState({
+            isLoggedin: true
+          });
+        });
+    } catch (err) {
+      return err.message;
+    }
+  };
+
+  render() {
+    const { email, password, isLoggedin } = this.state;
+    const { from } = this.props.location.state || {
+      from: { pathname: "/home" }
+    };
+
+    if (isLoggedin) {
+      return <Redirect to={from} />;
+    } else {
```

Comment:
no need for this else as return breaks out from the function

Reply:
comment resolved